src/cgame/cg_local.h

```diff
@@ -2050,6 +2050,20 @@ void CG_PositionRotatedEntityOnTag( refEntity_t *entity, const refEntity_t *pare
                                     qhandle_t parentModel, const char *tagName );
 void CG_TransformSkeleton( refSkeleton_t *skel, const vec_t scale );
 
+team_t CG_Team(const entityState_t &es);
```

Comment:
It would be good to have a comment about whether these functions always relate to the client, or if they take on the role of the watched player in case of a spectator client.

Reply:
Good point. They take the role of the watched player, for HUD purposes I guess.

Reply:
I've added a comment and an alternative for those who could want it.